core/commands/config.go

```diff
@@ -24,7 +24,7 @@ type ConfigField struct {
 
 var ConfigCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "get and set IPFS config values",
+		Tagline: "get and set IPFS config values.",
```

Comment:
Somehow missed that.
Used regex /Tagline:.*"[[:lower:]]/ to fix them once for all.